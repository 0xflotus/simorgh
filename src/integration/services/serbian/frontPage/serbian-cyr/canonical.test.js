/**
 * @service serbian
 * @pathname /serbian/cyr
 */

import runCanonicalTests from '../../../../pages/frontPage/canonicalTests';

describe('Canonical serbian frontPage', runCanonicalTests);
