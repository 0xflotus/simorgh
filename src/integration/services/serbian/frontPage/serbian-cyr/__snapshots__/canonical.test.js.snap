// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Canonical serbian frontPage A11y I can see the skip to content link 1`] = `"Пређите на садржај"`;

exports[`Canonical serbian frontPage Analytics ATI 1`] = `"<img height=\\"1px\\" width=\\"1px\\" alt=\\"\\" style=\\"position:absolute\\" src=\\"https://logws1363.ati-host.net?s=598343&amp;s2=81&amp;p=serbiancyr.page&amp;x1=[urn:bbc:cps:631e99d6-c1c4-73b7-e050-17ac8045512e]&amp;x2=[responsive]&amp;x3=[news-serbian]&amp;x4=[sr-Cyrl]&amp;x7=[index-home]&amp;x8=[simorgh-nojs]&amp;x9=[Почетна+страна+-+BBC+News+на+српском]&amp;x11=[2018-01-19T14:09:41.000Z]&amp;x12=[2019-09-08T20:45:22.000Z]\\"/>"`;

exports[`Canonical serbian frontPage Footer I can click on the BBC branding and it would take me to the homepage 1`] = `"/serbian"`;

exports[`Canonical serbian frontPage Footer I can see the footer copyright and external linking text 1`] = `"© 2020 BBC. ББЦ није одговоран за садржај других сајтова. Прочитајте наша правила о линковању других сајтова."`;

exports[`Canonical serbian frontPage Header I can see the branding 1`] = `"BBC News, на српском"`;

exports[`Canonical serbian frontPage I can see Index Alsos 1`] = `"Повезано"`;

exports[`Canonical serbian frontPage SEO Canonical link 1`] = `"http://localhost:7080/serbian/cyr"`;

exports[`Canonical serbian frontPage SEO Dir attribute 1`] = `"ltr"`;

exports[`Canonical serbian frontPage SEO FB admins 1`] = `"100004154058350"`;

exports[`Canonical serbian frontPage SEO FB app ID 1`] = `"1609039196070050"`;

exports[`Canonical serbian frontPage SEO Heading level 1 1`] = `"BBC News, на српском - Главна страница"`;

exports[`Canonical serbian frontPage SEO Lang attribute 1`] = `"sr-Cyrl"`;

exports[`Canonical serbian frontPage SEO Linked data 1`] = `
Object {
  "@context": "http://schema.org",
  "@type": "WebPage",
  "image": Object {
    "@type": "ImageObject",
    "height": 576,
    "url": "https://news.files.bbci.co.uk/ws/img/logos/og/serbian.png",
    "width": 1024,
  },
  "mainEntityOfPage": Object {
    "@id": "http://localhost:7080/serbian/cyr",
    "@type": "WebPage",
    "name": "Почетна страна",
  },
  "publisher": Object {
    "@type": "NewsMediaOrganization",
    "logo": Object {
      "@type": "ImageObject",
      "height": 576,
      "url": "https://news.files.bbci.co.uk/ws/img/logos/og/serbian.png",
      "width": 1024,
    },
    "name": "BBC News на српском",
    "publishingPrinciples": "https://www.bbc.com/serbian/cyr/institutional-50173730",
  },
  "thumbnailUrl": "https://news.files.bbci.co.uk/ws/img/logos/og/serbian.png",
  "url": "http://localhost:7080/serbian/cyr",
}
`;

exports[`Canonical serbian frontPage SEO OG description 1`] = `"ББЦ на српском нуди ексклузиван садржај - аналитичко, истраживачкo и непристрасно извештавање у текстовима и видео прилозима прилагођеним и друштвеним мрежама."`;

exports[`Canonical serbian frontPage SEO OG image 1`] = `"https://news.files.bbci.co.uk/ws/img/logos/og/serbian.png"`;

exports[`Canonical serbian frontPage SEO OG image alt 1`] = `"ББЦ News на српском"`;

exports[`Canonical serbian frontPage SEO OG locale 1`] = `"sr-cyrl"`;

exports[`Canonical serbian frontPage SEO OG site name 1`] = `"BBC News на српском"`;

exports[`Canonical serbian frontPage SEO OG title 1`] = `"Главна страница - BBC News на српском"`;

exports[`Canonical serbian frontPage SEO OG type 1`] = `"website"`;

exports[`Canonical serbian frontPage SEO OG url 1`] = `"http://localhost:7080/serbian/cyr"`;

exports[`Canonical serbian frontPage SEO Page title 1`] = `"Главна страница - BBC News на српском"`;

exports[`Canonical serbian frontPage SEO Robots meta tag 1`] = `"noodp,noydir"`;

exports[`Canonical serbian frontPage SEO Twitter card 1`] = `"summary_large_image"`;

exports[`Canonical serbian frontPage SEO Twitter creator 1`] = `"@bbcnasrpskom"`;

exports[`Canonical serbian frontPage SEO Twitter description 1`] = `"ББЦ на српском нуди ексклузиван садржај - аналитичко, истраживачкo и непристрасно извештавање у текстовима и видео прилозима прилагођеним и друштвеним мрежама."`;

exports[`Canonical serbian frontPage SEO Twitter image alt 1`] = `"ББЦ News на српском"`;

exports[`Canonical serbian frontPage SEO Twitter image src 1`] = `"https://news.files.bbci.co.uk/ws/img/logos/og/serbian.png"`;

exports[`Canonical serbian frontPage SEO Twitter site 1`] = `"@bbcnasrpskom"`;

exports[`Canonical serbian frontPage SEO Twitter title 1`] = `"Главна страница - BBC News на српском"`;
